src/Core/LeanCode.Components.Startup/LeanStartup.cs

```diff
@@ -11,6 +11,7 @@ public abstract class LeanStartup
 {
     protected IConfiguration Configuration { get; }
     protected ILogger Logger { get; }
+    protected virtual bool CloseAndFlushLogger { get; set; } = true;
```

Comment:
```suggestion
    protected virtual bool CloseAndFlushLogger { get; } = true;
```

Otherwise, someone from the outside can "just" set it (just the way you do in tests, which is not the aim IMO).